index.d.ts

```diff
@@ -23,6 +25,7 @@ export interface PackageJson {
   peerDependencies?: Dependencies;
   bundleDependencies?: Dependencies;
   bundledDependencies?: Dependencies;
+  optionalDependencies?: ObjectOfStrings;
```

Comment:
`optionalDependencies` was only declared on `ManifestVersion` before. Now it's declared on `PackageJson` and `ManifestVersion` uses `Pick<PackumentVersion, "optionalDependencies">`.